### PR TITLE
  RDM-9053: Postgress 11 upgrade

### PR DESCRIPTION
**Pull request by Thor-tech-of-metal**
RDM-9053: Postgress 11 upgrade
    
               [RDM-9053] (https://tools.hmcts.net/jira/browse/RDM-9053).

### CODE REVIEW

File: k8s/demo/common/ccd/definition-store-api.yaml [hunk 1]
```diff
@@ -19,12 +19,24 @@ spec:
...
       environment:
         IDAM_USER_URL: https://idam-web-public.demo.platform.hmcts.net
         DEFINITION_STORE_DB_OPTIONS: "?sslmode=require&gssEncMode=disable"
         DEFINITION_STORE_DB_MAX_POOL_SIZE: 32
+        DEFINITION_STORE_DB_NAME: ccd_definition_store-v11
+        DEFINITION_STORE_DB_HOST: ccd-definition-store-api-postgres-db-v11-demo.postgres.database.azure.com
+        DEFINITION_STORE_DB_USERNAME: ccd@ccd-definition-store-api-postgres-db-v11-demo
         DEFINITION_STORE_S2S_AUTHORISED_SERVICES: ccd_data,ccd_gw,ccd_admin,jui_webapp,pui_webapp,aac_manage_case_assignment
     global:
       environment: demo
```
Comment (Thor-tech-of-metal) [inline k8s/demo/common/ccd/definition-store-api.yaml:42]: Other microservices are going to be added once we prove connectivity to the new DB-v11 using v11 credentials

File: k8s/demo/common/ccd/definition-store-api.yaml [hunk 2]
```diff
@@ -19,12 +19,24 @@ spec:
...
       replicas: 2
+      keyVaults:
+        ccd:
+          secrets:
+            - definition-store-api-POSTGRES-PASS-V11
+            - ccd-definition-s2s-secret
+            - ccd-ELASTIC-SEARCH-URL
+            - ccd-ELASTIC-SEARCH-PASSWORD
+            - storage-account-primary-connection-string
+            - AppInsightsInstrumentationKey
```
Comment (mario-paniccia) [inline k8s/demo/common/ccd/definition-store-api.yaml:30]: do you know if you need to redefine all the secrets?
Or can we omit `AppInsightsInstrumentationKey, ccd-definition-s2s-secret, storage-account-primary-connection-string` and those will be picked up by default from `values.yml`?

Comment (Thor-tech-of-metal) [reply k8s/demo/common/ccd/definition-store-api.yaml:30]: I had checked that , it does not apply this logic for vault secrets.  given that secrets is the  key map value  we need to overwrite everything.    It looks likes it is like that.

**Review by nathan-clark — APPROVED** [review]

**Review by nathan-clark — DISMISSED** [review]

**Review by mario-paniccia — DISMISSED** [review]
👍

**Review by mario-paniccia — APPROVED** [review]
👍

**Review by hemantt — APPROVED** [review]

**Pull request merged**